cdap-data-fabric/src/main/java/co/cask/cdap/data/stream/service/StreamHandler.java

```diff
@@ -177,6 +182,18 @@ public void destroy(HandlerContext context) {
   }
 
   @GET
+  @Path("/")
+  public void listStreams(HttpRequest request, HttpResponder responder,
+                          @PathParam("namespace-id") String namespaceId) throws Exception {
+    List<StreamSpecification> specificationList = streamAdmin.listStreams(new NamespaceId(namespaceId));
```

Comment:
nit: call it specifications, since the datatype already suggests that its a list?